cypress/integration/v2/blocks/cta.spec.js

```diff
@@ -76,3 +79,404 @@ function switchDesign() {
+		cy.collapse( 'Description' )
+		cy.adjust( 'Typography', {
+			[ `Font Family` ]: 'Sans-Serif',
+			[ `Size` ]: 23,
+			[ `Weight` ]: '200',
+			[ `Transform` ]: 'capitalize',
+			[ `Line-Height` ]: {
+				value: 49,
+				unit: 'px',
+			},
+			[ `Letter Spacing` ]: 1.9,
+		} )
+		cy.adjust( 'Size', 1.3, { unit: 'em' } )
+		cy.adjust( 'Description Color', '#333333' ).assertComputedStyle( {
+			'.ugb-cta__description': {
+				[ `font-size` ]: '1.3em',
+				[ `font-weight` ]: '200',
+				[ `text-transform` ]: 'capitalize',
+				[ `letter-spacing` ]: '1.9px',
+				[ `color` ]: '#333333',
+				[ `line-height` ]: '49px',
+			},
+		} )
+		assertAligns( 'Align', '.ugb-cta__description' )
+
+		// Test Button options
+		cy.collapse( 'Button' )
+		cy.adjust( 'Button Color Type', 'single' )
+		cy.adjust( 'Button Color', '#ff0000' )
+		cy.adjust( 'Text Color', '#ffffff' ).assertComputedStyle( {
+			'.ugb-button': {
+				[ `background-color` ]: '#ff0000',
+			},
+			'.ugb-button--inner': {
+				[ `color` ]: '#ffffff',
+			},
+		} )
+		cy.adjust( 'Hover Effect', 'lift' )
+			.assertClassName( '.ugb-button', 'ugb--hover-effect-lift' )
+		cy.adjust( 'Hover Opacity', 0.6 )
+		cy.adjust( 'Hover Colors', {
+			[ `Button Color` ]: '#ffffff',
+			[ `Text Color` ]: '#000000',
+		} )
+		cy.adjust( 'Typography', {
+			[ `Font Family` ]: 'Serif',
+			[ `Size` ]: 19,
+			[ `Weight` ]: '200',
+			[ `Transform` ]: 'lowercase',
+			[ `Letter Spacing` ]: 2,
+		} ).assertComputedStyle( {
+			'.ugb-button--inner': {
+				[ `font-size` ]: '19px',
+				[ `font-weight` ]: '200',
+				[ `text-transform` ]: 'lowercase',
+				[ `letter-spacing` ]: '2px',
+			},
+		} )
+		cy.adjust( 'Button Size', 'medium' )
+			.assertClassName( '.ugb-button', 'ugb-button--size-medium' )
+		cy.adjust( 'Border Radius', 35 )
+		cy.adjust( 'Vertical Padding', 4 )
+		cy.adjust( 'Horizontal Padding', 27 )
+		cy.adjust( 'Shadow', 2 )
+			.assertClassName( '.ugb-button', 'ugb--shadow-2' )
+		cy.adjust( 'Opacity', 0.6 ).assertComputedStyle( {
+			'.ugb-button': {
+				[ `border-radius` ]: '35px',
+				[ `padding-top` ]: '4px',
+				[ `padding-bottom` ]: '4px',
+				[ `padding-left` ]: '27px',
+				[ `padding-right` ]: '27px',
+				[ `opacity` ]: '0.6',
+			},
+		} )
+		cy.waitFA()
+		cy.adjust( 'Icon', 'info' )
+		cy.adjust( 'Adv. Icon Settings', {
+			[ `Icon Size` ]: 18,
+			[ `Icon Position` ]: 'right',
+			[ `Icon Spacing` ]: 29,
+		} ).assertComputedStyle( {
+			'.ugb-button svg': {
+				[ `width` ]: '18px',
+				[ `height` ]: '18px',
+				[ `margin-left` ]: '29px',
+			},
+		} )
+		assertAligns( 'Align', '.ugb-button-container' )
+
+		// Test Effects
+		cy.collapse( 'Effects' )
+		cy.adjust( 'Hover Effect', 'lift' )
+			.assertClassName( '.ugb-cta__item', 'ugb--hover-lift' )
+
+		// Test Block Background
+		assertBlockBackground( '.ugb-cta', { viewport: 'Desktop' } )
+
+		// Test Top and Bottom Separator
+		assertSeparators( { viewport: 'Desktop' } )
+	} )
+}
+
+function responsiveAssert( mode ) {
```

Comment:
instead of naming the variable `mode`, better to name it viewport and do `{ viewport }` under options